cli/src/bin/phylum.rs

```diff
@@ -177,34 +175,26 @@ async fn handle_commands() -> CommandResult {
 
 async fn handle_ping(api: PhylumApi) -> CommandResult {
     print_user_success!("{}", api.ping().await?);
-    Ok(ExitCode::Ok.into())
+    Ok(ExitCode::Ok)
 }
 
 #[cfg(feature = "selfmanage")]
 async fn handle_update(matches: &ArgMatches, ignore_certs: bool) -> CommandResult {
     let res = update::do_update(matches.get_flag("prerelease"), ignore_certs).await;
     let message = res?;
     print_user_success!("{}", message);
-    Ok(ExitCode::Ok.into())
+    Ok(ExitCode::Ok)
 }
 
 fn handle_version(app_name: &str, ver: &str) -> CommandResult {
     print_user_success!("{app_name} (Version {ver}){LICENSE_BLURB}");
-    Ok(ExitCode::Ok.into())
+    Ok(ExitCode::Ok)
 }
 
 #[tokio::main]
 async fn main() {
     match handle_commands().await {
-        Ok(CommandValue::Action(action)) => match action {
-            Action::None => ExitCode::Ok.exit(),
-            Action::Warn => exit_warn("Project failed threshold requirements!"),
```

Comment:
This was the only use of `exit_warn`. It's dead code now, but isn't detected because it is (pointlessly) `pub`.

You can delete `exit_warn` and remove the `pub` for `exit_fail`.